test/commontransformations.jl

```diff
@@ -333,11 +345,11 @@
             trans = euler_rotation(0.1,0.2,0.3)
             x2 = Point(2.730537054338937,0.8047190852558106,2.428290466296628)
 
-            @test trans.t1.t1 == RotationXY(0.1)
-            @test trans.t1.t2 == RotationYZ(0.2)
-            @test trans.t2 == RotationZX(0.3)
+            #@test trans.t1.t1 == RotationXY(0.1)
+            #@test trans.t1.t2 == RotationYZ(0.2)
+            #@test trans.t2 == RotationZX(0.3)
```

Comment:
Sure works for me.  Can be worth adding a TODO to make it clear why a code block was commented out otherwise it sometimes ends up hanging around.